modules/collections/client/controllers/collections.controllers.js

```diff
@@ -1,139 +1,154 @@
+      // Open document directory in Document Manager.
+      $scope.goToDocument = function(doc) {
+        return '/p/' + $scope.project.code + '/docs?folder=' + doc.directoryID;
+      };
 
-      $scope.confirmMove = function(title, msg, moveFunc) {
-        var modalDocView = $uibModal.open({
-          animation: true,
-          templateUrl: 'modules/utils/client/views/partials/modal-confirm-generic.html',
-          controller: function($scope, $state, $uibModalInstance) {
-            var self = this;
-            self.title = title || 'Move document?';
-            self.question = msg || 'Are you sure?';
-            self.actionOK = 'Move';
-            self.actionCancel = 'Cancel';
-            self.ok = function() {
-              $uibModalInstance.close($scope.project);
-            };
-            self.cancel = function() {
-              $uibModalInstance.dismiss('cancel');
-            };
-          },
-          controllerAs: 'self',
-          scope: $scope,
-          size: 'md',
-          windowClass: 'modal-alert',
-          backdropClass: 'modal-alert-backdrop'
-        });
-        modalDocView.result.then(function(/* res */) {
-          moveFunc();
+      // Shorthand for transitioning to Edit. This is interchangeable with $state.reload() *if* we're already in Edit (not in Create).
+      $scope.goToEdit = function() {
+        $state.transitionTo('p.collection.edit', { projectid: project.code, collectionId: collection._id }, {
+          reload: true, inherit: false, notify: true
         });
-      };
+      }
 
-      var goToList = function() {
+      // Shorthand for transitioning to List.
+      $scope.goToList = function() {
         $state.transitionTo('p.collection.list', { projectid: project.code }, {
           reload: true, inherit: false, notify: true
         });
-      };
+      }
 
-      var reloadEdit = function() {
-      // want to reload this screen, do not catch unsaved changes (we are probably in the middle of saving).
-        $scope.allowTransition = true;
-        $state.reload();
+      // Called after using the Document Manager modal.
+      $scope.updateDocuments = function(updatedDocuments, docType) {
+        var alternate = docType == 'main' ? 'other' : 'main';
+        var duplicates = $scope.findDuplicates(updatedDocuments, $scope.modifiedDocuments[alternate]);
+        if (duplicates.length) {
+          // If duplicates are found...
+          var errMsg = '';
+          _.forEach(duplicates, function(doc) {
+            errMsg += ('<br/> - ' + doc.displayName);
+            updatedDocuments = _.without(updatedDocuments, doc);
+            // Remove them from the updated documents list, and alert the user.
+          })
+          AlertService.error('The following document(s) already exist in "' + (docType == 'main' ? 'Related' : 'Main') + ' Documents" and have not been added:' + errMsg, 7000, false);
```

Comment:
Nit.: there is no need to pass the **false** parameter to the AlertService, as it is optional and will default to that value when omitted.

Reply:
Removed